docs/can-canjs/can-api.md

```diff
@@ -0,0 +1,289 @@
+@page api API Docs
+@parent canjs 2
+@outline 0
+@package ../../package.json
+@templateRender <% %>
+@description
+Welcome to the CanJS docs!
```

Comment:
Okay, I've updated the description. Let me know if you don't like the new one. I can try to come up with alternatives.